contrib/devtools/security-check.py

```diff
@@ -116,6 +117,25 @@ def check_ELF_CONTROL_FLOW(binary) -> bool:
         return True
     return False
 
+def check_ELF_FORTIFY(binary) -> bool:
+
+    # bitcoin-util does not currently contain any fortified functions
+    if 'Bitcoin Core bitcoin-util utility version ' in binary.strings:
+        return True
+
+    chk_funcs = set()
+
+    for sym in binary.imported_symbols:
+        match = re.search(r'__[a-z]*_chk', sym.name)
+        if match:
+            chk_funcs.add(match.group(0))
+
+    # ignore stack-protector and bdb
+    chk_funcs.discard('__stack_chk')
+    chk_funcs.discard('__db_chk')
```

Comment:
Bit unfortunate that this list has to be maintained, but I can't think of a better way either.

Reply:
Yea, it's a bit annoying. At least post-BDB, it'll just be "ignore the stack protector".